man/systemd-sbsign.xml

```diff
@@ -109,6 +120,16 @@
         <xi:include href="version-info.xml" xpointer="v258"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--secure-boot-database=PK|KEK|db|dbx</option></term>
+
+        <listitem><para>Specify the secure boot signature database that should be generated and signed with
+        the <command>sign-secure-boot-database</command> command. Takes one of <varname>PK</varname>,
+        <varname>KEK</varname>, <varname>db</varname> or <varname>dbx</varname>.</para>
```

Comment:
should say in which format these files should be

Reply:
ah, these aren't files, but enums? we always use lowercase for that. please do so here too

Reply:
or hmm, these are efi var names? maybe say "Takes one of the strings …"

Reply:
how does uefi spec call these? probably not "signature databases", because they contain certs and hashes, but no sigs...

Reply:
They actually do call these signature databases: https://uefi.org/specs/UEFI/2.11/32_Secure_Boot_and_Driver_Signing.html#signature-database

Reply:
weird. but ok then